test/e2e/harness.go

```diff
@@ -177,6 +185,19 @@ func (h *harness) deployContractAndRequireSuccess(t *testing.T, keyPair *keys.Ed
 	require.EqualValues(t, codec.EXECUTION_RESULT_SUCCESS, dcExResult, "expected deploy contract to succeed")
 }
 
+func (h *harness) deployJSContractAndRequireSuccess(t *testing.T, keyPair *keys.Ed25519KeyPair, contractName string, contractBytes ...[]byte) {
```

Comment:
Should move to another package.
@electricmonk can help you generalize the e2e harness so that it can be used from another package